definitions/npm/jasmine_v2.4.x/flow_all/jasmine_v2.4.x.js

```diff
@@ -60,6 +60,21 @@ type JasmineClockType = {
   mockDate(date: Date): void;
 }
 
+declare type JasmineMatcherResult = {
+  pass: boolean;
+  message?: string | () => string;
+}
+
+declare type JasmineMatcherStruct = {
+  compare<T: any>(actual: T, expected: T): JasmineMatcherResult;
+}
+
+declare type JasmineMatcher = (utils?: mixed, customEqualityTesters?: mixed) => JasmineMatcherStruct
```

Comment:
It's not entirely clear to me what `utils` and `customEqualityTesters` are from reading the Jasmine docs.  The most that I could tell is that `customEqualityTesters` is an array, but I'm not sure what of.  `utils` is a mystery.